benchmarks/python/test_groupnorm_fwd.py

```diff
@@ -121,6 +127,24 @@ def test_groupnorm_fwd_nvf_benchmark(
         run_benchmark(benchmark, fd.execute, [x, weight, bias])
 
 
+@pytest.mark.parametrize("size", generate_input_sizes(dims=4))
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+def test_groupnorm_fwd_thunder_benchmark(
+    benchmark,
+    size: tuple,
+    dtype: torch.dtype,
+):
+    clear_cuda_cache()
+    N, C, H, W = size
+    x = torch.randn(size, device="cuda", dtype=dtype, requires_grad=True)
+    weight = torch.randn(C, device="cuda", dtype=dtype, requires_grad=True)
+    bias = torch.randn(C, device="cuda", dtype=dtype, requires_grad=True)
+    num_groups = get_n_goups(C)
+    # thunder compiled model
+    groupnorm_fwd_jit = thunder.jit(groupnorm_fwd, nv_enable_bookend=False)
```

Comment:
Do you want to test group norm inference forward or training forward? The difference between the two is in the number of saved intermediates. `disable_torch_autograd_support=True` would stop Thunder from implicitly generating the backward trace and the forward will be the inference forward.

thunder.jit's default executors might change, I suggest using an explicit list of executors:
```suggestion
    from thunder.executors.nvfuserex import nvfuserex
    groupnorm_fwd_jit = thunder.jit(groupnorm_fwd, nv_enable_bookend=False, executors=[nvfuserex])
```

Reply:
Good point! Changed to explicitly using nvFuser.

In this case, the output seems to be controlled only by `requires_grad=False/True` and `disable_torch_autograd_support` doesn't seem to influence the output. I tested the following 4 cases:
- case-1, `requires_grad=True for x, weight, bias ` and `disable_torch_autograd_support=False` -->3 outputs, `groupNorm(x), mean(x), rsqrt(var)`
- case-2, `requires_grad=False for x, weight, bias ` and `disable_torch_autograd_support=False` --> 1 output, `groupNorm(x)`
- case-3, `requires_grad=True for x, weight, bias ` and `disable_torch_autograd_support=True`--> 3 outputs, `groupNorm(x), mean(x), rsqrt(var)` (same to case-1)
- case-4, `requires_grad=False for x, weight, bias ` and `disable_torch_autograd_support=True`--> 1 output, `groupNorm(x)`(same to case-2)

The additional output of ` mean(x), rsqrt(var)` won't cause a large influence on performance, but we should be consistant for different tests and we can also make it a test para `@pytest.mark.parametrize("require_grad", [False, True])`. Will check with the team about which version to use. @kevinstephano

Reply:
talked with @kevinstephano, we want to prioritize training forward